chalicelib_fourfront/checks/wfr_checks.py

```diff
@@ -132,26 +132,31 @@ def md5run_status(connection, **kwargs):
     if lab:
         query += '&lab.display_title=' + lab
     # The search
-    res = ff_utils.search_metadata(query, key=my_auth)
+    res = ff_utils.search_metadata(query, key=my_auth, is_generator=True)
     if not res:
```

Comment:
Also maybe naive, but does the generator returned in the previous line evaluate differently here? I am not sure what the generator will be if there are no results.

Reply:
Another great catch!  Thanks!